src/main/java/org/folio/services/ftp/FtpUploadService.java

```diff
@@ -110,47 +91,61 @@ private Handler<AsyncResult<Object>> asyncResultHandler(Promise<String> promise,
     };
   }
 
-  public Future<String> upload(Context ctx, String filename, String content) {
+  private Handler<AsyncResult<String>> asyncResult(Promise<String> promise) {
+    return result -> {
+      if (result.succeeded()) {
+        logger.info("Success upload to FTP");
+        promise.complete(result.result());
+      } else {
+        logger.error("Failed upload to FTP");
+        String message = Optional.ofNullable(result.cause())
+          .map(Throwable::getMessage)
+          .orElse(result.cause().getMessage());
+        logger.error(message);
```

Comment:
The same change needed as in similar comment regarding logging

Reply:
fixed